.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+Thanks for trying out Zero to JupyterHub!
+
+If you're having issues with your deployment, it is
+really helpful if you include the contents of your
+`config.yaml` file with your issue. We've put
+a place below where you can paste this in. Thanks!
```

Comment:
ah, I see what you mean by ssl keys. We should ask people to retract any secret info, including but not limited to:

1. any special keys under auth
2. proxy.secretToken
3. hub.cookieSecret